devops/charts/bg-monitoring/values.yaml

```diff
@@ -1,6 +1,67 @@
+timezone: Europe/Berlin
+#Monitoring server configuration
 app:
+  enabled: true #enable app deployment,secret,service,pvc
+  replicas: 1
+  image:
+    stream: true #enable annotation to look for stream image name
+    stream_name: bg-monitoring-server 
+    stream_tag: "latest"
+    pullPolicy: Always
+    name: bg-monitoring-server #Use this if image.stream set to false
+    tag: latest #Use this if image.stream set to false
   route:
-    host: dev-bg-monitoring-server.cloud.adorsys.de
+    enabled: true #Enable external route to app
+    host: >-
+      {{ .Release.Name }}-monitoring-server.cloud.adorsys.de
```

Comment:
This is not equivalent change. The resulting host will be "bg-monitoring-dev-monitoring-server.cloud.adorsys.de".

Reply:
Equivalent for what changes? it depends of a release name